contracts/Positions/PositionalMarketManager.sol

```diff
@@ -63,6 +64,12 @@ contract PositionalMarketManager is Initializable, ProxyOwned, ProxyPausable, IP
 
     bool public needsTransformingCollateral;
 
+    uint public timeframe;
```

Comment:
not quite clear on the purpose of this variable from the naming alone, maybe rename to "timeframeBuffer"

Reply:
As the timeframe is deducted and added to proposed strike date and then truncated to midnight on a given date, I think it makes more sense to use int128 for this and have values represent amount of days the markets need be apart, e.g. 1, 2 or 3 days